### PR TITLE
EASY-1943 Publish DANS BagIt Profile

### PR DESCRIPTION
**Pull request by janvanmansum**
@DANS-KNAW/easy for review

- [x] Update `DRAFT` status and publication date
- [x] Register DOI at DataCite: https://doi.org/10.17026/dans-z52-ybfe should point to v0 document.

Related PR: https://github.com/DANS-KNAW/easy-schema/pull/70

### CODE REVIEW

File: versions/0.0.0.md [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+
+5. There MUST NOT be more than one `file` element corresponding to a payload file and every payload file MUST be described
+   by a `file` element.
+
+6. Each `file` element MUST contain at least one `dcterms:format` element containing the [Media Type]
+   of the file described.
+
+7. Each `file` element MAY contain any number of other [DC] or [DCTERMS] elements describing the file. It MUST NOT contain elements
+   from other namespaces, unless permitted by the [DANS bag file metadata schema] schema.
+   <!-- Skip if XSD is used -->
```
Comment (rvanheest) [inline versions/0.0.0.md]: is this 'comment' visible in the rendered markdown on GitHub?

File: versions/0.0.0.md [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+
+Introduction
+------------
+
+### Version
+* Document version: 0.0.0
+* Publication date: 2019-04-15
+
+### Status
+Status of this document is DRAFT.
```
[the remark this answers is not in the document]
Comment (janvanmansum) [reply versions/0.0.0.md]: Let's update this and the Publication date as the last step, just to be sure.

File: versions/0.0.0.md (hunk 1 above)
Comment (janvanmansum) [reply versions/0.0.0.md]: No, we might also remove it. It's not really important, just a reminder that the next rules are already checked by the XSD.

File: versions/0.0.0-licenses.txt [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+ http://www.gnu.org/licenses/gpl-3.0.en.html
+ http://www.gnu.org/licenses/lgpl-3.0.txt
+ http://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html
+ http://www.mozilla.org/en-US/MPL/2.0/FAQ/
+ http://www.ohwr.org/attachments/735/CERNOHLv1_1.txt
+ http://www.ohwr.org/attachments/2388/cern_ohl_v_1_2.txt
+ http://www.ohwr.org/projects/cernohl/wiki
+ http://www.tapr.org/ohl.html (alias for http://www.tapr.org/TAPR_Open_Hardware_License_v1.0.txt, *deprecated*)
+ http://www.tapr.org/TAPR_Open_Hardware_License_v1.0.txt
```
Comment (lindareijnhoudt) [inline versions/0.0.0-licenses.txt]: addition:
https://opendatacommons.org/licenses/by/1-0/index.html

what is the procedure for adding licenses here?

File: versions/0.0.0.md [hunk 4]
```diff
@@ -0,0 +1,365 @@
...
+5. There MUST NOT be more than one `file` element corresponding to a payload file and every payload file MUST be described
+   by a `file` element.
+
+6. Each `file` element MUST contain at least one `dcterms:format` element containing the [Media Type]
+   of the file described.
+
+7. Each `file` element MAY contain any number of other [DC] or [DCTERMS] elements describing the file. It MUST NOT contain elements
+   from other namespaces, unless permitted by the [DANS bag file metadata schema] schema.
+   <!-- Skip if XSD is used -->
+
```
Comment (lindareijnhoudt) [inline versions/0.0.0.md]: new rule 3.2.8 from EASY-2029, do you want it here already? or in a separate PR?

3.2.8
Each `file` element MAY contain either one dcterms:accessRights (DEPRECATED) or files:accessibleToRights and/or files:visibleToRights (RECOMMENDED). The values of these elements are limited to ANONYMOUS, RESTRICTED_REQUEST and NONE. It is recommended to make files visible to all, by selecting `files:visibleToRights=ANONYMOUS`

File: versions/0.0.0.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (janvanmansum) [reply versions/0.0.0.md]: done

**Comment by janvanmansum** [comment]
Problems with my fork; creating new PR and closing this one.

**Pull request closed without merging**